pet-bot.js

```diff
@@ -12,14 +14,27 @@ client.on("ready", () => {
 
 client.on("message", async (msg) => {
 	if (msg.content.startsWith("pet")) {
-		let user;
+		
+		let user, avatar, id, avatarUrl;
+
 		if (msg.mentions.users.first()) {
 			user = msg.mentions.users.first();
-		} else {
+			avatar = user.avatar;
+			if (!avatar) {
+				return msg.channel.send(`<@${user.id}> no image no pets!`);
+			}
+			avatarUrl = `https://cdn.discordapp.com/avatars/${id}/${avatar}.png`;
+		} 
+		else {
 			user = msg.author;
+			searchQuery = msg.content.split(" ")[1];
+
+			let url = `https://api.unsplash.com/search/photos?query=${searchQuery}&client_id=${process.env.PHOTO_API_TOKEN}`
+			let response = await axios.get(url)
+
+			avatarUrl = response.data.results[0].urls.small;
 		}
-		const avatar = user.avatar;
-		const id = user.id;
+
 		if (!avatar) {
```

Comment:
doing `pet obama` 
would break it here right?
Since the avatar variable is undefined.

Reply:
Oh man, I have pushed unsaved code and the profile URL has `{id}` instead of `{user.id}`.
I have pushed the code, try locally and lemme know. 🔆